proto/addressmetadata.proto

```diff
@@ -1,39 +1,74 @@
 syntax = "proto3";
 package models;
 
-message Header {
-    string Name = 1;
-    string Value = 2;
+// Specifies the revocation method
+enum RevocationMethod {
```

Comment:
"Method" may not be a good term for it here, method may be a reserved word in some languages and it doesn't sound too expressive.

"RevocationType" or "RevocationScheme" however scheme collides with signature scheme below.

Reply:
Let's inline this into the AddressMetadata.

Reply:
I thought about this, but putting it outside the AddressMetadata scope allows it to be used in the actual `Revocation` message as well as in the Revocation Clause?